### ISSUE TITLE
Wrap new AdSense setup stories with proper backdrop markup

### ISSUE THREAD

**Issue by felixarntz** [google/site-kit-wp#5177]
Several AdSense setup stories for the refined setup flow (V2) have been introduced recently as part of implementing the respective logic and UI (e.g. https://google.github.io/site-kit-wp/storybook/main/?path=/story/modules-adsense-components-setup-v2-setupmain--ad-blocker).

There's an issue with all of these stories in that they are not wrapped with the boxed layout that is present in the plugin, making them in fact not look like they look in-product. While this may not seem necessary from an engineering perspective, one of the purposes of Storybook is to provide a visual reference, which here is not accurate - so we should fix this. Setup flow stories for other modules (e.g. https://google.github.io/site-kit-wp/storybook/main/?path=/story/modules-analytics-setup-setupformga4transitional--without-existing-tag) are properly wrapped, and the AdSense ones should be too.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* All Storybook stories in _AdSense > Components > Setup > V2 > SetupMain_ (i.e. starting with https://google.github.io/site-kit-wp/storybook/main/?path=/story/modules-adsense-components-setup-v2-setupmain--ad-blocker) should be wrapped with the regular white box in setup and have the Site Kit header bar above them.
    * In other words, the wrapping should look the same way that other module setup stories look like (e.g. https://google.github.io/site-kit-wp/storybook/main/?path=/story/modules-analytics-setup-setupformga4transitional--without-existing-tag).

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

* N/A

**Comment by felixarntz** [comment]
Actually, looks like this was already fixed as part of #4763 since `main` was branched off, it is correct in `develop`.
